tests/unit/harvesters/test_civic_assertions.py

```diff
@@ -0,0 +1,109 @@
+"""Test CIViC source"""
+import pytest
+from metakb.harvesters.civic import CIViC
+from civicpy import civic as civicpy
+
+
+@pytest.fixture(scope='module')
+def civic():
+    """Create a list of genes."""
+    civicpy.load_cache(on_stale='ignore')
+    c = CIViC()
+    return c
+
+
+@pytest.fixture(scope='module')
+def aid40():
+    """Create a fixture for AID40 assertion."""
+    return {
```

Comment:
Over time, these fixtures will drift as the source service changes.   Consider managing them outside of test source